app.js

```diff
@@ -1,5 +1,62 @@
+import { randomWare } from './utilities.js';
+
+const userGuess = document.getElementById('user-guess');
+
+const guessButton = document.getElementById('guess-button');
+
+const remainingGuesses = document.getElementById('rem-guesses');
+
+const clue = document.getElementById('higher-lower');
+
+const mysteryNumber = document.getElementById('mystery-number');
+
+// console.log(userGuess, guessButton, remGuesses, clue, mysteryNumber);
+
+
+// page loads randomWare needs to run once.
+// Need to hide number. 
+
+
+const randomNumber = randomWare(); 
+
+// console.log(randomNumber)
+
+mysteryNumber.textContent = randomNumber;
+
+// console.log(mysteryNumber.textContent)
+
+// butoon is clicked 
+// remaining guesses goes down
+// userinput compared to mystery number
+
+// if they guess right:
 // import functions and grab DOM elements
 
 // initialize state
+let remGuesses = 4;   
+// set event listeners to update state and DOM;
+
+guessButton.addEventListener('click', () => {
+    remGuesses--;
+    compareTwoNum(randomNumber, userGuess) 
+});
+
+
+function compareTwoNum(randomNumber, userGuess) { 
```

Comment:
We talked yesterday and I hope my explanations made since at the end of the day. Feel free to finish it and submit it when you can. You were so close!